docs/supported_apis/dataframe_supported.rst

```diff
@@ -103,15 +110,16 @@ default to pandas.
 +----------------------------+---------------------------+------------------------+----------------------------------------------------+
 | ``dot``                    | `dot`_                    | Y                      |                                                    |
 +----------------------------+---------------------------+------------------------+----------------------------------------------------+
-| ``drop``                   | `drop`_                   | Y                      |                                                    |
+| ``drop``                   | `drop`_                   | Y                      | **Omnisci**:``P``, ``D`` for row drop              |
 +----------------------------+---------------------------+------------------------+----------------------------------------------------+
 | ``droplevel``              | `droplevel`_              | Y                      |                                                    |
 +----------------------------+---------------------------+------------------------+----------------------------------------------------+
 | ``drop_duplicates``        | `drop_duplicates`_        | D                      |                                                    |
 +----------------------------+---------------------------+------------------------+----------------------------------------------------+
-| ``dropna``                 | `dropna`_                 | Y                      |                                                    |
+| ``dropna``                 | `dropna`_                 | Y                      | **Omnisci**:``P``, same as ``drop``,               |
+|                            |                           |                        | also ``D`` for ``thresh`` and ``axis`` parameters  |
 +----------------------------+---------------------------+------------------------+----------------------------------------------------+
-| ``dtypes``                 | `dtypes`_                 | Y                      |                                                    |
+| ``dtypes``                 | `dtypes`_                 | Y                      | **Omnisci**:``Y`` ?                                |
```

Comment:
`dtypes` is fully supported, no need for a question tag

Reply:
Updated